src/App.tsx

```diff
@@ -15,71 +15,88 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component {
+  state: Readonly<State> = {
+    selectedGood: 'Jam is',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  handleClearButton = () => {
+    this.setState({ selectedGood: 'No goods' });
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  handleAddGood = (add: string) => {
+    this.setState({ selectedGood: `${add} is` });
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  render() {
+    return (
+      <main className="section container">
+        <h1 className="title is-flex is-align-items-center">
+          {`${this.state.selectedGood} selected`}
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+          {this.state.selectedGood !== 'No goods' && (
```

Comment:
if you use 'No goods' several times, should create variable for it (default value or something like that)